packit.yaml

```diff
@@ -9,7 +9,8 @@ actions:
   # https://github.com/packit/sandcastle/pull/92
   # https://medium.com/the-node-js-collection/node-js-memory-management-in-container-environments-7eb8409a74e8
   create-archive:
-  - env NODE_OPTIONS=--max-old-space-size=500 make dist-gzip
+  - env NODE_OPTIONS=--max-old-space-size=750 make all
+  - env NODE_OPTIONS=--max-old-space-size=750 make dist-gzip
```

Comment:
FTR, 300 seems to be the magic number. Big enough to let the build succeed without "out of heap" errors, small enough for 768 MiB RAM.